### ISSUE TITLE
It's impossible to make the yes/no fields read-only

### ISSUE THREAD

**Issue by bennygill** [pods-framework/pods#4961]
## Issue Overview
It's impossible to make the yes/no fields read-only.

Restricting the field by selecting the `Make field "Read Only" in UI` has no effect.

## Expected Behavior
The yes/no field should become greyed out when `Make field "Read Only" in UI` is selected. See here: https://screenshots.firefox.com/ctgWdV8n5fi7Ygvh/www.datafreedoms.com

## Current Behavior
The field does not become "read-only" because it can still be changed. It should be greyed out so you can see it's current state (yes or no) but are unable to change it. See here: https://screenshots.firefox.com/fRd4CfbLfTP9Re8d/www.datafreedoms.com

## Steps to Reproduce (for bugs)

1. Add a yes/no custom field to a post
2. Under the advanced tab select `Make field "Read Only" in UI` 
3. Go to the post and you will see it is not read-only, because you can change it.

## WordPress Environment

<details>
```
Debug Information

WordPress Version: 4.9.6

PHP Version: 7.0.30-1~dotdeb+8.1

MySQL Version: 5.6.39

Server Software: Apache/2.4.10 (Debian)

Your User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:60.0) Gecko/20100101 Firefox/60.0

Session Save Path: /var/lib/php/sessions

Session Save Path Exists: Yes

Session Save Path Writeable: Yes

Session Max Lifetime: 1440

Opcode Cache:

    Apc: Yes
    Memcached: No
    OPcache: Yes
    Redis: No

Object Cache:

    APC: Yes
    APCu: Yes
    Memcache: No
    Memcached: Yes
    Redis: No

WPDB Prefix: wp_

WP Multisite Mode: No

WP Memory Limit: 40M

Pods Network-Wide Activated: No

Pods Install Location: /home/43307-85855.cloudwaysapps.com/jwzqnqwkyc/public_html/wp-content/plugins/pods/

Pods Tableless Mode Activated: No

Pods Light Mode Activated: No

Currently Active Theme: Pro – Child Theme

Currently Active Plugins:

    Include Me: 1.1.5
    Pods - Custom Content Types and Fields: 2.7.3
    Pods - Custom Points Score:
    Search by Algolia – Instant & Relevant results: 2.11.1
```
</details>

**Comment by codente** [comment]
I can replicate this.  It also seems that you aren't able to set a "default value" so that if I wanted it checked/Yes by default, I can't.

**Comment by jimtrue** [comment]
If you set the default value to 1 then it will check 'yes' by default, but that also means it's always checked.

**Comment by codente** [comment]
Yep, that's what I assumed would happened.  But I didn't consider the fact that I had already saved this posts before adding in that default value.  

So guess the only issue here is that it's not read only and can be changed. :)

**Comment by atanas-angelov-dev** [comment]
Link to my proposed PR: #5164

**Comment by sc0ttkclark** [comment]
PR looks good
